tikzplotlib/_axes.py

```diff
@@ -53,26 +53,24 @@ def __init__(self, data, obj):  # noqa: C901
             xlabel = _common_texification(xlabel)
             self.axis_options.append(f"xlabel={{{xlabel}}}")
             if xrotation != 0:
-                self.axis_options.append(
-                    "xlabel style={{rotate={}}}".format(xrotation - 90)
-                )
+                self.axis_options.append("xlabel style={{rotate={xrotation - 90)}}}")
```

Comment:
you missed the 'f' and a stray ')' remains

Reply:
Thanks!